canonical-web-and-design/react-components
# Remove custom SearchAndFilter styling

## Done

- Remove custom styling from SearchAndFilter, use class names provided by Vanilla instead

Fixes: #404

## QA

### QA steps

- Check the demo of SearchAndFilter component:
  - https://react-components-424.demos.haus/?path=/story/search-and-filter--default-story
- Make sure it works and looks as expected
  - try different screen widths
  - try all the functionality: clicking on chips, typing search, overflowing chips, etc

### Storybook

To see rendered examples of all react-components, run:

```shell
yarn start
```

### QA in your project

from `react-components` run:

```shell
yarn build
npm pack
```

Install the resulting tarball in your project with:

```shell
yarn add <path-to-tarball>
```
